### PR TITLE
Add Village de la Justice article

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/translations/en/media.mdx [hunk 1]
```diff
@@ -16,6 +16,24 @@
 
 ---
 
+### Village de la Justice
+
+<p className="text__smallcaps">September 27, 2021</p>
+
+> Whatsapp, Google, Facebook et autres sont devenus notre quotidien. Nous y passons notre temps, et pourtant nous ne les connaissons pas. Pas juridiquement parlant en tout cas. Pour ce faire il faudrait prendre le temps de lire leurs Conditions Générales d’Utilisation, mais, encore plus, de les re-lire tant leurs changements sont fréquents et loin d’être anodins pour les utilisateurs, et tant ces CGU en disent long sur le service proposé.
```
Comment (MattiSG) [inline src/translations/en/media.mdx:23]: Translated with DeepL:

```suggestion
> Whatsapp, Google, Facebook and the like have become our daily lives. We spend our time on them, and yet we don't know them. At least not legally speaking. To do so, we should take the time to read their Terms of Use, but even more so, to re-read them as their changes are frequent and far from trivial for users, and as these ToS say a lot about the service offered. Open Terms Archive was created to remedy this lack of knowledge, which contributes to the imbalance of power between digital giants and users.
```

File: src/translations/en/media.mdx [hunk 2]
```diff
@@ -16,6 +16,24 @@
...
+
+<a
+  target="_blank"
+  rel="noopener"
+  className="float__right"
+  title="Un outil open source facilitant l’opposabilité des CGU des géants du numérique."
```
Comment (MattiSG) [inline src/translations/en/media.mdx:29]:
```suggestion
  title="An open source tool facilitating the enforceability of the ToS of digital giants."
```

File: src/translations/en/media.mdx [hunk 3]
```diff
@@ -16,6 +16,24 @@
...
+  href="https://www.village-justice.com/articles/open-terms-archive,40252.html"
+>
+  <Button type="secondary">Read</Button>
```
Comment (MattiSG) [inline src/translations/en/media.mdx:32]:
```suggestion
  <Button type="secondary">Read (in French)</Button>
```

File: src/translations/fr/media.mdx [hunk 4]
```diff
@@ -16,6 +16,24 @@
 
 ---
 
+### Village de la Justice
+
+<p className="text__smallcaps">27 septembre 2021</p>
+
+> Whatsapp, Google, Facebook et autres sont devenus notre quotidien. Nous y passons notre temps, et pourtant nous ne les connaissons pas. Pas juridiquement parlant en tout cas. Pour ce faire il faudrait prendre le temps de lire leurs Conditions Générales d’Utilisation, mais, encore plus, de les re-lire tant leurs changements sont fréquents et loin d’être anodins pour les utilisateurs, et tant ces CGU en disent long sur le service proposé.
```
Comment (MattiSG) [inline src/translations/fr/media.mdx:23]: I feel like the most important part has been left out 🤔 

```suggestion
> Whatsapp, Google, Facebook et autres sont devenus notre quotidien. Nous y passons notre temps, et pourtant nous ne les connaissons pas. Pas juridiquement parlant en tout cas. Pour ce faire il faudrait prendre le temps de lire leurs Conditions Générales d’Utilisation, mais, encore plus, de les re-lire tant leurs changements sont fréquents et loin d’être anodins pour les utilisateurs, et tant ces CGU en disent long sur le service proposé. Open Terms Archive a été créé pour remédier à cette méconnaissance qui participe au rapport de force déséquilibré entre géants du numérique et utilisateurs.
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Makes me wonder if we would not benefit from using the title of the article as the title of the quote instead of just the media name 🤔

File: src/translations/en/media.mdx [hunk 5]
```diff
@@ -16,6 +16,24 @@
...
+> Whatsapp, Google, Facebook and the like have become our daily lives. We spend our time on them, and yet we don't know them. At least not legally speaking. To do so, we should take the time to read their Terms of Use, but even more so, to re-read them as their changes are frequent and far from trivial for users, and as these ToS say a lot about the service offered. Open Terms Archive was created to remedy this lack of knowledge, which contributes to the imbalance of power between digital giants and users.
+
+<a
+  target="_blank"
+  rel="noopener"
+  className="float__right"
+  title="An open source tool facilitating the enforceability of the ToS of digital giants."
+  href="https://www.village-justice.com/articles/open-terms-archive,40252.html"
+>
+  <Button type="secondary">Read (in French)</Button>
```
Comment (MattiSG) [inline src/translations/en/media.mdx:32]: This should be applied on all of the links towards French articles 🙂

**Review by MattiSG — APPROVED** [review]

File: src/translations/en/media.mdx (hunk 5 above)
Comment (clementbiron) [reply src/translations/en/media.mdx:50]: Yes, i will do it in a different pull request ;)

**Pull request merged**